tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -1029,10 +1029,9 @@ def _cudnn_rnn(inputs,
     args["time_major"] = time_major
     outputs, output_h, output_c, _, _ = gen_cudnn_rnn_ops.cudnn_rnnv3(**args)
   elif time_major is False:
-    max_time = inputs.get_shape().as_list()[1]
-    batch_size = inputs.get_shape().as_list()[0]
-    lengths = np.repeat(max_time, batch_size)
-    sequence_lengths = ops.convert_to_tensor(lengths.astype(np.int32))
+    batch_size = array_ops.shape(inputs)[0]
+    max_time = array_ops.shape(inputs)[1]
+    sequence_lengths = array_ops.fill([batch_size], max_time)
```

Comment:
Could you add relevant tests?

Reply:
I think I have already added tests for this feature (with time_major parameter). My original commit has (in cudnn_rnn_ops_test.py), which force uses to provide both variable_seq_lengths and time_major.
```
if not variable_seq_lengths and not time_major:	
      self.skipTest("Batch major not supported")
```
But now I remove these lines (see commit 9c27326b98f975dd04908defd9c6925e9263aa78). So it has the tests on this branch (time_major but no variable_seq_length)

Reply:
The tests didn't not cover dynamic shape input, thats why the previous commit's problem wasn't caught.

Reply:
OIC. Could you please point me to some example code in other ops unittest about adding the tf.placeholder and feed_dict?

Reply:
No worries. I found some examples. Will update it.